core/services/ocr2/plugins/ocr2keeper/evm21/registry_check_pipeline.go

```diff
@@ -126,10 +131,13 @@ func (r *EvmRegistry) verifyLogExists(upkeepId *big.Int, p ocr2keepers.UpkeepPay
 	// if log block number is populated, check log block number and block hash
 	if logBlockNumber != 0 {
 		h, ok := r.bs.queryBlocksMap(logBlockNumber)
+		// if this block number/hash combo exists in block subscriber, this block and tx still exists on chain and is valid
```

Comment:
similarly can we expand on the comment on why we ignore block subscriber's hash when it existed

Reply:
done